_includes/all_figures.html

```diff
@@ -53,3 +53,15 @@
 <p><img alt="GitHub Repository After First Push" src="../fig/github-repo-after-first-push.svg" /></p>
 <hr/>
 <p><img alt="The Conflicting Changes" src="../fig/conflict.svg" /></p>
+<hr/>
```

Comment:
The `</p>` tags are highlighted in red because they are within the incomplete `<img />` tags.
Please replace `</p>` with ` /></p>`.

Reply:
Thank you.